CS_Ed_Week/CS_Ed_Week.md

```diff
@@ -128,14 +154,32 @@ A workshop for getting young people interested in computer science.
 - To print the entire contents of a list, simply supply the name of list as the argument to print()
 - To print an specific element of a list, supply the name of the list, followed by its index, to print()
   - `pets = ["dog", "cat", "bird"]`
-  - `print(pets)`
-  - `print(pets[2])`
+  - `print(pets) # ['dog', 'cat', 'bird']`
+  - `print(pets[2]) # ['bird']`
+
+### Nested lists
+
+- Python also supports *nested* list.
+- Python allows you to store *any* type in a list, meaning that even lists can be in lists.
+- This essentially means that you can put loops inside of list.
+- The main reason you would want this would be if you want to hold relational data in the same place.
+- It looks just like a normal list: 
+  - `nested_list = [[1,3,5],[3,2,6],4,"5",True]`
+- The way you would call this data would be:
+  - `nested_list[0][0]`
+- It works by desending in scope. The first one selects the table, and then the 2nd one selects the data from that table.
+- It's even possible to repeat this infinitly! Lists inside lists!
+- I would highly suggest not mixing up nested lists and lists with other types though, as if you index the list when it isn't a list type, it will fail.
+- Example:
+  - `nested_list = [['Hello', 5, True], ['Goodbye', 0, False]]`
+  - `print(nested_list) # [['Hello', 5, True], ['Goodbye', 0, False]]`
+  - `print(nested_list[0]) # ['Hello', 5, True]`
+  - `print(nested_list[0][3]) # True`
+
 
 __Exercises__: 
-- Create a guest list for a party with at least three people. Print an individualized message to each member.
-- One of the guests can't make it to dinner, so you need to find someone else to invite. Modify your guest list accordingly.
-- You found a bigger dinner table, so you can invite three more people. Add a guest to the beginnig, middle, and end of your list.
-- Print new messages for these new guests.
+- Create a list of 'guest' members coming to a party.
```

Comment:
I think we should keep the previous 4 exercises you deleted, because it relates lists to a real world situation. The exercise about nested lists we should keep though.